.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ orbs:
 parameters:
   action:
     type: enum
-    enum: [default, bump, paywall_tester_release, generate_revenuecatui_snapshots]
+    enum: [default, bump, paywall_tester_release, generate_revenuecatui_snapshots, deploy_snapshot]
```

Comment:
Since the term "snapshot" is pretty ambiguous (e.g. snapshot tests, snapshot releases), maybe we can name this action `deploy_snapshot_release`? (Or `publish_snapshot_release`, but I see we're using "deploy" in more places in this file.)